docs/styles/spec/file-input-spec.less

```diff
@@ -20,19 +19,5 @@
         &:extend(hx-file-control > input[type="file"]:active ~ label > hx-file-input.hxBtn.hxTertiary);
       }
     }
-
-    hx-file-input.mock-hover {
-      &.hxBtn {
-        &:extend(.hxBtn:hover);
-      }
-
-      &.hxBtn.hxPrimary {
-        &:extend(.hxBtn.hxPrimary:hover);
-      }
-
-      &.hxBtn.hxTertiary {
-        &:extend(.hxBtn.hxTertiary:hover);
-      }
-    }
```

Comment:
These specific specs were migrated to SCSS. This file is still being loaded and will need migrated in a separate PR.